skillmap/src/styles/graphnode.css

```diff
@@ -18,4 +18,13 @@
 .graph-status-icon {
     font-family: Icons;
     font-size: 1rem;
+}
+
+.graph-activity {
+    cursor: pointer;
```

Comment:
Oh, just noticed, the status icon && graph icon need this too as they're also selectable

Reply:
I put it on the outer <g>, so it should apply to all of them